aws/table_aws_route53_traffic_policy.go

```diff
@@ -0,0 +1,261 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+)
+
+func tableAwsRoute53TrafficPolicy(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_route53_traffic_policy",
+		Description: "AWS Route53 Traffic Policy",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"id", "version"}),
+			Hydrate:           getTrafficPolicy,
+			ShouldIgnoreError: isNotFoundError([]string{"NoSuchTrafficPolicy"}),
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listTrafficPolicies,
+		},
+		Columns: awsColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name that you specified when traffic policy was created.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "id",
+				Description: "The ID that Amazon Route 53 assigned to a traffic policy when it was created.",
```

Comment:
```suggestion
				Description: "The ID that Amazon Route53 assigned to a traffic policy when it was created.",
```
should not be `route53` instead of `route 53`?